qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/AzureBackupAcceptanceIT.java

```diff
@@ -0,0 +1,108 @@
+import io.camunda.zeebe.broker.system.configuration.backup.BackupStoreCfg.BackupStoreType;
+import io.camunda.zeebe.qa.util.cluster.TestApplication;
+import io.camunda.zeebe.qa.util.cluster.TestCluster;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.qa.util.testcontainers.AzuriteContainer;
+import io.camunda.zeebe.test.util.testcontainers.ContainerLogsDumper;
+import java.util.Map;
+import org.agrona.CloseHelper;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+/**
+ * Acceptance tests for the backup management API. Tests here should interact with the backups
+ * primarily via the management API, and occasionally assert results on the configured backup store.
+ *
+ * <p>The tests run against a cluster of 2 brokers and 1 gateway, no embedded gateways, two
+ * partitions and replication factor of 1. This allows us to test that requests are correctly fanned
+ * out across the gateway, since each broker is guaranteed to be leader of a partition.
+ *
+ * <p>NOTE: this does not test the consistency of backups, nor that partition leaders correctly
+ * maintain consistency via checkpoint records. Other test suites should be set up for this.
+ */
+@Testcontainers
+@ZeebeIntegration
+final class AzureBackupAcceptanceIT implements BackupAcceptance {
+  @Container private static final AzuriteContainer AZURITE_CONTAINER = new AzuriteContainer();
+  private static final String CONTAINER_NAME = RandomStringUtils.randomAlphabetic(10).toLowerCase();
+
+  @RegisterExtension
+  @SuppressWarnings("unused")
+  final ContainerLogsDumper logsWatcher =
+      new ContainerLogsDumper(() -> Map.of("azurite", AZURITE_CONTAINER));
+
+  // cannot auto start, as we need azurite to be started before we can configure the brokers
+  @TestZeebe(autoStart = false)
+  private final TestCluster cluster =
+      TestCluster.builder()
+          .withBrokersCount(2)
+          .withGatewaysCount(1)
+          .withReplicationFactor(1)
+          .withPartitionsCount(2)
+          .withEmbeddedGateway(false)
+          .withNodeConfig(this::configureNode)
+          .build();
+
+  private AzureBackupStore store;
+
+  @BeforeEach
+  void beforeEach() {
+    final AzureBackupConfig config =
+        new AzureBackupConfig.Builder()
+            .withConnectionString(AZURITE_CONTAINER.getConnectString())
+            .withContainerName(CONTAINER_NAME)
+            .build();
+    store = new AzureBackupStore(config);
+    final BlobServiceClient blobServiceClient = AzureBackupStore.buildClient(config);
+    blobServiceClient.createBlobContainerIfNotExists(CONTAINER_NAME);
```

Comment:
:question: Isn't that something that the store does for us? Or am I misremembering?